app/tasks.py

```diff
@@ -404,41 +406,33 @@ def get(self):
                 logging.info("Timeout occurs in datastore")
                 self.add_task_for_repo(
                     self.repo, self.task_name(), self.ACTION)
-            finally:
-                self._notify(count_of_unreviewed_notes)
 
-    def _notify(self, count_of_unreviewed_notes):
+    def _maybe_notify(self, count_of_unreviewed_notes):
         # TODO: Response should be modified
-        if self._is_ok_to_notify(count_of_unreviewed_notes):
-            self.send_mail(self._get_admin_user_email(),
+        if self._should_modify(count_of_unreviewed_notes):
+            self.send_mail(self._get_unreviewed_notes_email(),
                            subject="PersonFinder: Please review your notes",
-                           body="{0} notes are not reviewed".format(
-                               count_of_unreviewed_notes))
+                           body="%d notes are not reviewed" %(
+                               count_of_unreviewed_notes,
+                           ))
             return True
         else:
             return False
 
-    def _is_ok_to_notify(self, count_of_unreviewed_notes):
-        email = self._get_admin_user_email()
-        if not email:
+    def _should_modify(self, count_of_unreviewed_notes):
```

Comment:
s/_should_modify/_should_notify/

(In case you don't know the notation above, it means "replace _should_modify with _should notify", following sed syntax.)